docs/changehistory/NextVersion.md

```diff
@@ -120,6 +120,9 @@ Several changes were made in the @bentley/ui-* packages.
 Some components in @bentley/ui-core were deprecated in favor of components in @itwinui-react.
 A few constructs were deprecated in @bentley/ui-core package with alternatives elsewhere.
 
+The @bentley/ui-* packages are now dependent on React version 17.
+**Applications using the ui packages must update React 17.**
```

Comment:
Are there any docs or guides to move from React 16 -> 17 that they put out?

Reply:
There's not much to the update from React 16 to React 17. I could add a link to the blog post they published for the release -- would that work for you?

Reply:
Yup, that's exactly what I was thinking. Just to point people in the right direction.

Reply:
Done!